pkg/buildah/manifest/manifest.go

```diff
@@ -51,3 +50,19 @@ func (c *CommandArgs) isManifestCmd() (bool, error) {
 
 	return true, err
 }
+
+func manifestCommands(sources, targets []string) []*exec.Cmd {
+	cmds := []*exec.Cmd{}
+
+	for _, target := range targets {
+		cmds = append(cmds, exec.Command(common.BuildahCmd, manifestCommand, "create", storageDriverOptionVfs, target))
+
+		for _, source := range sources {
+			cmds = append(cmds, exec.Command(common.BuildahCmd, manifestCommand, "add", storageDriverOptionVfs, target, source))
```

Comment:
Does having an image layer ref from a different source than the target work if you run the image later?

Reply:
Yes it does work, I modified the [test .drone.yml](https://github.com/yuzhouliu9/web-app-go/blob/main/.drone.yml#L110-L117) with source from a python web-app-1. The target is the web-app-go, and this pipeline ran and deployed successfully, and apps work.

I think since the target doesn't actually exist during creation time, there is no layer pulled. So the target is just a name placeholder I think.

Reply:
That was also my interpretation of how target worked.  It is where the stitched together manifest goes.

Reply:
Thanks, as discussed, the sources and targets will now just be the image tags. The registry and repo and settings will be used to construct the full path.

Here is a [working example](https://github.com/yuzhouliu9/web-app-go/blob/main/.drone.yml#L110-L115), manifest commands ran successfully and single image was successfully deployed on both amd64 and arm64.

Reply:
I like that a lot more than before.